OutOfSchool/OutOfSchool.WebApi.Tests/Services/ProviderServicesTests/PublicProviderServiceTests.cs

```diff
@@ -88,5 +92,41 @@ public async Task UpdateStatus_WhenProviderIdIsNonExistent_ReturnsNull()
         Assert.IsNull(result);
     }
 
+    [Test]
+    public async Task UpdateStatus_WhenDtoIsValid_UpdateAddedInChangesLog()
+    {
+        // Arrange
+        var expectedCallTimes = Times.Once;
+
+        var provider = ProvidersGenerator.Generate();
+        provider.Status = ProviderStatus.Pending;
+        var dto = new ProviderStatusDto
+        {
+            ProviderId = provider.Id,
+            Status = ProviderStatus.Approved,
+        };
+
+        providersRepositoryMock.Setup(r => r.GetById(dto.ProviderId))
+            .ReturnsAsync(provider);
+        providersRepositoryMock.Setup(r => r.UnitOfWork.CompleteAsync())
+            .ReturnsAsync(It.IsAny<int>());
+
+        changesLogServiceMock.Setup(
+                s => s.AddEntityChangesToDbContext(
+                    It.IsAny<Provider>(),
+                    It.IsAny<string>()))
+            .Returns(1);
```

Comment:
(Test clarity) Please use specific `Provider` and `userId` instead of `It.IsAny`. Here in `.Setup()` and below in `.Verify()`.

Reply:
Fixed

Reply:
It could be simpler ))

Instead of 
```
        changesLogServiceMock.Setup(
                s => s.AddEntityChangesToDbContext(
                    It.Is<Provider>(p => p.Id == dto.ProviderId),
                    It.Is(fakeUser.Id, StringComparer.Ordinal)))
            .Returns(1);
```
just use
```
        changesLogServiceMock.Setup(s => s.AddEntityChangesToDbContext(dto.ProviderId, fakeUser.Id))
            .Returns(1);
```

Reply:
> It could be simpler ))
> 
> Instead of
> 
> ```
>         changesLogServiceMock.Setup(
>                 s => s.AddEntityChangesToDbContext(
>                     It.Is<Provider>(p => p.Id == dto.ProviderId),
>                     It.Is(fakeUser.Id, StringComparer.Ordinal)))
>             .Returns(1);
> ```
> 
> just use
> 
> ```
>         changesLogServiceMock.Setup(s => s.AddEntityChangesToDbContext(dto.ProviderId, fakeUser.Id))
>             .Returns(1);
> ```

Got it, thanks)
But this PR can be closed, on daily meeting we discussed that this is not what exactly should be added to `ChangesLogs`

Sorry for bothering with review